storage/innobase/include/mem0mem.ic

```diff
@@ -597,17 +597,9 @@ mem_strdupl(
 	ulint		len)	/*!< in: length of str, in bytes */
 {
 	char*	s = static_cast<char*>(ut_malloc_nokey(len + 1));
-<<<<<<< HEAD
 	if (!s) return NULL;
 
 	s[len] = 0;
-||||||| a9b0c712de3
```

Comment:
Any reason why this has been resolved in favor of our changes, not upstream?

Reply:
The reasoning is our change is the same as theirs but safer.

If `s` is null and `len > 0`, theirs would reference a `NULL` ptr but not ours.

Reply:
OK, indeed looks safer this way